app/scripts/migrations/068.test.js

```diff
@@ -0,0 +1,2491 @@
+        extensionId: null,
+      },
+      'https://matcha.xyz': {
+        iconUrl: 'https://matcha.xyz/favicon.ico',
+        name: 'Matcha',
+        origin: 'https://matcha.xyz',
+        extensionId: null,
+      },
+      'https://medium.com': {
+        iconUrl: 'https://miro.medium.com/1*m-R_BkNf1Qjr1YbyOIJY2w.png',
+        name: 'Medium',
+        origin: 'https://medium.com',
+        extensionId: null,
+      },
+      'https://metamask.github.io': {
+        iconUrl: null,
+        name: 'MetaMask < = > Ledger Bridge',
+        origin: 'https://metamask.github.io',
+        extensionId: null,
+      },
+      'https://opensea.io': {
+        iconUrl: 'https://opensea.io/static/images/favicon/32x32.png',
+        name: 'OpenSea',
+        origin: 'https://opensea.io',
+        extensionId: null,
+      },
+      'https://sentry.io': {
+        iconUrl:
+          'https://s1.sentry-cdn.com/_static/aa8be2d373cced45f56eeb8d0af3a476/sentry/images/favicon.png',
+        name: 'Sentry',
+        origin: 'https://sentry.io',
+        extensionId: null,
+      },
+      'https://wesbos.com': {
+        iconUrl: 'https://wesbos.com/favicon.svg',
+        name: 'Wes Bos',
+        origin: 'https://wesbos.com',
+        extensionId: null,
+      },
+      'https://www.ethdenver.com': {
+        iconUrl:
+          'https://images.squarespace-cdn.com/content/v1/614b990e002ad146a5478e8b/339800af-d572-4cb7-8415-861a9a1aa7cc/favicon.ico',
+        name: 'ETHDenver 2022',
+        origin: 'https://www.ethdenver.com',
+        extensionId: null,
+      },
+      'https://www.google.com': {
+        iconUrl: null,
+        name: 'Redirecting',
+        origin: 'https://www.google.com',
+        extensionId: null,
+      },
+      'https://www.howtogeek.com': {
+        iconUrl: 'https://www.howtogeek.com/public/favicon.ico',
+        name: 'How-To Geek',
+        origin: 'https://www.howtogeek.com',
+        extensionId: null,
+      },
+      'https://www.linkedin.com': {
+        iconUrl: 'https://static-exp1.licdn.com/sc/h/1bt1uwq5akv756knzdj4l6cdc',
+        name: 'LinkedIn',
+        origin: 'https://www.linkedin.com',
+        extensionId: null,
+      },
+      'https://www.npmjs.com': {
+        iconUrl:
+          'https://static.npmjs.com/b0f1a8318363185cc2ea6a40ac23eeb2.png',
+        name: 'npm',
+        origin: 'https://www.npmjs.com',
+        extensionId: null,
+      },
+      'https://www.youtube.com': {
+        iconUrl: null,
+        name: 'YouTube',
+        origin: 'https://www.youtube.com',
+        extensionId: null,
+      },
+      'https://xdai.io': {
+        iconUrl: 'https://xdai.io/favicon.ico',
+        name: 'Burner',
+        origin: 'https://xdai.io',
+        extensionId: null,
+      },
+    },
+  };
+}
+
+function getOldState() {
+  return {
+    FooController: { a: 'b' }, // just to ensure it's not touched
+    PermissionsController: {
+      domains: {
+        'https://faucet.metamask.io': {
```

Comment:
Is it necessary to have such a large fixture? This makes it difficult to tell what has changed.

Reply:
Hm, probably not. I cut down on it significantly here: cd4d3a5d2